manifests/w/wargaming/GameCenter/21.03.00.5224/wargaming.GameCenter.yaml

```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Wargaming.GameCenter
+PackageVersion: 21.03.00.5224
```

Comment:
```suggestion
PackageVersion: 21.3.0.5224
```

Reply:
Resubmitted with correct folder structure and updated package version.